subprojects/testing-native/src/main/java/org/gradle/nativeplatform/test/xctest/tasks/InstallXCTestBundle.java

```diff
@@ -80,7 +80,14 @@ void install() throws IOException {
             "#!/bin/sh"
                 + "\nAPP_BASE_NAME=`dirname \"$0\"`"
                 + "\nXCTEST_LOCATION=`xcrun --find xctest`"
-                + "\nexec \"$XCTEST_LOCATION\" \"$@\" \"$APP_BASE_NAME/" + bundleDir.getName() + "\""
+                + "\n"
+                + "\nARGUMENTS=(\"$@\")"
+                + "\nTEST_FILTER=\"\""
+                + "\nif [ \"${#ARGUMENTS[@]}\" -eq 1 ]; then"
+                + "\n    TEST_FILTER=\"-XCTest ${ARGUMENTS[0]}\""
+                + "\nfi"
```

Comment:
This code is to unify both XCTest implementation:
 - On Linux: `$ ./AppTest <filter>`
 - On macOS: `$ xctest -XCTest <filter> AppTest.xctest`

Reply:
couldn't we do this in the other code?  e.g., pass `-XCTest <filter>` when on macOS and don't use `-XCTest` on Linux?

Reply:
I changed it.